engine-types/src/types.rs

```diff
@@ -89,14 +90,20 @@ impl Wei {
     pub fn checked_add(self, other: Self) -> Option<Self> {
         self.0.checked_add(other.0).map(Self)
     }
+
+    pub fn checked_mul(self, other: Self) -> Option<Self> {
```

Comment:
Multiplying units of tokens (Wei in this case) does not make sense. The result would have units of `Money^2` which is meaningless (there is not a reasonable notion of "money area" as far as I know).

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/244/commits/08be2514183780684699a6fc27d8cbb25f58598e